source/views/building-hours/list.tsx

```diff
@@ -20,51 +25,85 @@ const styles = StyleSheet.create({
 	},
 })
 
-type Props = TopLevelViewPropsType & {
-	now: Moment
-	loading: boolean
-	onRefresh: () => unknown
-	buildings: Array<{title: string; data: BuildingType[]}>
+function useBuildingHours() {
+	return useFetch<BuildingType[]>(API('/spaces/hours'), {
+		headers: {accept: 'application/json'},
+	})
 }
 
-export function BuildingHoursList(props: Props): JSX.Element {
+const emptyList: ReadonlyArray<never> = []
+
+function groupBuildings(
+	buildings: Array<BuildingType> | void,
+	favorites: Array<string>,
+) {
+	if (!buildings) {
+		return emptyList
+	}
+
```

Comment:
I think we may need to either change the return type for what group returns or what emptyList is returning.

At line 43 we see the following:

```log
Unexpected any. Specify a different type
```

The return signature for `groupBuildings` is

```js
readonly never[] | {  title: string;
    data: BuildingType[];
}[]
```